docs/content/functioneel/zaken_api.md

```diff
@@ -0,0 +1,104 @@
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een zaak toevoegen, gegevens van een zaak wijzigen, wijzigingen aanbrengen in de relatie van een zaak tot betrokkenen en/of objecten dan wel één of meer gegevens van zaken en genoemde relaties opvragen.                                                                                                            |
+| Voorwaarde     | 1. Het zaaktype van de zaak moet aanwezig en geldig zijn (resource: zaaktypen in API: ZTC). <br>2.  Een te relateren Betrokkene moet aanwezig zijn (resource: [...] in API:     [...]). <br>3.  Een te relateren Object moet aanwezig zijn (resource: [...] in API: [...]). <br>4.  Bij het toevoegen van een zaak mag de (functionele) Identificatie meegegeven worden; deze dient uniek te zijn. |
+| Gevolg         | De zaak wordt, indien sprake is van valide gegevens, toegevoegd aan (bij een POST) dan wel gewijzigd in (PUT, PATCH) de Zaakregistratiecomponent dan wel worden (bij GET) gegevens van de gevonden zaken beschikbaar gesteld. Indien bij een POST geen functionele identificaties zijn meegeleverd dan wordt deze door de ZRC bepaald. |
+| Gegevens       | Objecttype ZAAK, inclusief relaties naar Zaaktype, Betrokkene (incl. relatieklasse ROL) en OBJECT (incl. relatieklasse ZAAKOBJECT). <br>Zie de volgende tabel voor de gegevens van deze resource |
+| Specificaties  | <https://ref.tst.vng.cloud/zrc/api/v1/schema/#tag/zaken>                                                                                                                                                                                                                                                                               |
+| Samenhang      | Update van Zaak.Einddatum vindt plaats vanuit de resource Statussen.                                                                                                                                                                                                                                                                   |
+| Bijzonderheden | -  De operatie DELETE wordt niet ondersteund: zaken kunnen niet verwijderd worden. <br>-   Een zaak moet tenminste één betrokkene in een Rol hebben. <br>-   De relatie naar Zaaktype (cq. het zaaktype van de zaak) kan niet gewijzigd worden. Indien het zaaktype onjuist blijkt te zijn, dient de zaak beeindigd te worden en een nieuwe zaak (met het juiste zaakype) toegevoegd te worden. <br>-   De Einddatum van de zaak kan alleen opgevraagd worden. Muteren hiervan gebeurt in de resource: Statussen. |
```

Comment:
> De operatie DELETE wordt niet ondersteund: zaken kunnen niet verwijderd worden.

Ik zou deze eerste deelzin weglaten, want technisch. Functioneel voldoet dat "zaken niet verwijderd kunnen worden". Dit is ook meer in lijn met de andere bijzonderheden.

Technisch staat er inderdaad geen DELETE bij de zaken resource.

Reply:
Ik heb dat toch laten staan om ondubbelzinnig de link naar de technische uitvoering te leggen. Ook omdat helemaal boven in staat dat alle operaties voor alle resources gelden, tenzij anders vermeld.

Reply:
Ik denk dat HTTP methods enkel voor developers relevant zijn, dus zou zelf ook opteren om enkel te stellen dat 'Zaken kunnen niet verwijderd worden.', maar ik ga er ook geen drama om maken, dus jij hebt laatste woord wat mij betreft. :)

Reply:
Ik heb de zin nu omgedraaid.